### PR TITLE
[rdsca] New extractor

### LINKED ISSUE #1611: Add support for rds.ca
Hi,

Could you add support for http://www.rds.ca ? 
Thanks.

Example url:

$ youtube-dl --verbose http://www.rds.ca/vid%C3%A9os/un-voyage-positif-3.877934
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'http://www.rds.ca/vid%C3%A9os/un-voyage-positif-3.877934']
[debug] youtube-dl version 2013.10.17
[debug] Python version 3.3.2 - Linux-3.11.5-1-ARCH-x86_64-with-arch
[debug] Proxy map: {}
WARNING: Falling back on generic information extractor.
[generic] un-voyage-positif-3.877934: Downloading webpage
[generic] un-voyage-positif-3.877934: Extracting information
ERROR: Unsupported URL: http://www.rds.ca/vid%C3%A9os/un-voyage-positif-3.877934; please report this issue on https://yt-dl.org/bug . Be sure to call youtube-dl with the --verbose flag and include its complete output. Make sure you are using the latest version; type  youtube-dl -U  to update.
Traceback (most recent call last):
  File "/usr/lib/python3.3/site-packages/youtube_dl/YoutubeDL.py", line 353, in extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python3.3/site-packages/youtube_dl/extractor/common.py", line 117, in extract
    return self._real_extract(url)
  File "/usr/lib/python3.3/site-packages/youtube_dl/extractor/generic.py", line 173, in _real_extract
    raise ExtractorError(u'Unsupported URL: %s' % url)
youtube_dl.utils.ExtractorError: Unsupported URL: http://www.rds.ca/vid%C3%A9os/un-voyage-positif-3.877934; please report this issue on https://yt-dl.org/bug . Be sure to call youtube-dl with the --verbose flag and include its complete output. Make sure you are using the latest version; type  youtube-dl -U  to update.

### PR DESCRIPTION
**Pull request by fstirlitz**
This should resolve #1611.

### CODE REVIEW

File: youtube_dl/extractor/rdsca.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+from .common import InfoExtractor
+from ..utils import (
+    parse_iso8601,
+    url_basename,
+)
+
+
+class RDScaIE(InfoExtractor):
+    IE_NAME = 'RDS.ca'
+    _VALID_URL = r'http://(?:www\.)?rds\.ca/videos/(?P<id>.*)'
```
Comment (dstftw) [inline youtube_dl/extractor/rdsca.py]: Looks like all the URLs satisfy the following format: http://www.rds.ca/videos/.../<display_id>.<id>. `_VALID_URL` should be fixed correspondingly or test with another URL schema should be provided.

File: youtube_dl/extractor/rdsca.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    _TESTS = [{
+        'url': 'http://www.rds.ca/videos/football/nfl/fowler-jr-prend-la-direction-de-jacksonville-3.1132799',
```
Comment (dstftw) [inline youtube_dl/extractor/rdsca.py]: There should be md5 of the file's first 10k.

File: youtube_dl/extractor/rdsca.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        'info_dict': {
```
Comment (dstftw) [inline youtube_dl/extractor/rdsca.py]: `id` and `display_id` should be tested as well.

File: youtube_dl/extractor/rdsca.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            "ext": "mp4",
```
Comment (dstftw) [inline youtube_dl/extractor/rdsca.py]: Use quotes consistently - only a single or only a double. Use different kind of quotes only if there is no way to use same kind of quotes.

File: youtube_dl/extractor/rdsca.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            "title": "Fowler Jr. prend la direction de Jacksonville",
+            "description": "Dante Fowler Jr. est le troisième choix du repêchage 2015 de la NFL. ",
+            "timestamp": 1430397346,
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = url_basename(url)
```
Comment (dstftw) [inline youtube_dl/extractor/rdsca.py]: As already mentioned `id` and `display_id` should be extracted.

File: youtube_dl/extractor/rdsca.py (hunk 1 above)
Comment (fstirlitz) [inline youtube_dl/extractor/rdsca.py]: Well... yes?

File: youtube_dl/extractor/rdsca.py (hunk 5 above)
Comment (fstirlitz) [inline youtube_dl/extractor/rdsca.py]: There seems to be no such thing as an "ID", apart from the URL itself.

**Pull request merged**